packages/client/src/components/app/forms/S3Upload.svelte

```diff
@@ -78,12 +83,15 @@
   }
 
   const upload = async () => {
+    const processedFileKey = processStringSync(key, {
+      [$component.id]: fieldState,
```

Comment:
This means the only context available is itself, but we need to add the normal context in here too don't we? Since you have access to all normal bindings when configuring this setting in the builder.

So we just need to pull in normal context as well:
```js
const context = getContext("context")
...
const processedFileKey = processStringSync(key, {
  ...$context,
  [$component.id]: fieldState,
})
```